src/js/containers/publish-page.js

```diff
@@ -26,18 +26,18 @@ class PublishPage extends Component {
   }
 
   onPointRemove( point ) {
-    this.props.rescindPoint(point._id);
+    this.props.rescindPoint( point._id );
   }
 
   onPublish() {
     this.props.publishPoints();
   }
 
   render() {
-    const clear = (
-    <ClearIcon style={ { fontSize: 'inherit', margin: '0px 0.1em' } }
-      color='red' />
-    );
+    // const clear = (
+    // <ClearIcon style={ { fontSize: 'inherit', margin: '0px 0.1em' } }
+    //   color='red' />
+    // );
```

Comment:
If we dont need this should we remove it?

Reply:
I wasn't really sure if it was something that they couldn't get working but you can see they had it commented out bellow. I figured I would leave it for now, we can clean it up later if needed.